modules/local/sentieon/tnscope.nf

```diff
@@ -0,0 +1,52 @@
+process SENTIEON_TNSCOPE {
+    tag	"$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path fasta
+    path fai
+
+    output:
+    tuple val(meta), path("*_TNscope_MTcalls_unfiltered.vcf.gz")        , emit: vcf
+    tuple val(meta), path("*_TNscope_MTcalls_unfiltered.vcf.gz.tbi")    , emit: vcf_index
+    path "versions.yml"                                                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def interval      = task.ext.args ?: ''
+    def call_settings = task.ext.args2 ?: ''
```

Comment:
Are there recommendations on how to name this type of things? (e.g. `args` or `args2` instead of `call_settings`)

Reply:
I'm not sure about it, seen different variable names in different modules!